Class issues when using aut-0.17.0-fatjar.jar

**Describe the bug**

`java.lang.NoClassDefFoundError` and `java.lang.NoClassDefFoundError` errors are produced when using `aut-0.17.0-fatjar.jar`. Looks like a potential class conflict in the fatjar, but not when the using `--packages`.

**To Reproduce**
1. `spark-shell --jars /home/nruest/Downloads/aut-0.17.0-fatjar.jar`
2. Load libraries, and `RecordLoader.loadArchives`:
```
Welcome to
      ____              __
     / __/__  ___ _____/ /__
    _\ \/ _ \/ _ `/ __/  '_/
   /___/ .__/\_,_/_/ /_/\_\   version 2.4.0
      /_/
         
Using Scala version 2.11.12 (Java HotSpot(TM) 64-Bit Server VM, Java 1.8.0_181)
Type in expressions to have them evaluated.
Type :help for more information.

scala> :paste
// Entering paste mode (ctrl-D to finish)

import io.archivesunleashed._
import io.archivesunleashed.matchbox._
val r = RecordLoader.loadArchives("/home/nruest/tmp/auk/75/3552/warcs/ARCHIVEIT-3552-DAILY-IEDLRH-20130212071008-00001-wbgrp-crawl052.us.archive.org-6680.warc.gz", sc)

// Exiting paste mode, now interpreting.

java.lang.NoClassDefFoundError: net/sourceforge/cobertura/coveragedata/TouchCollector
  at io.archivesunleashed.package$RecordLoader$.__cobertura_init(package.scala)
  at io.archivesunleashed.package$RecordLoader$.<clinit>(package.scala)
  ... 49 elided
Caused by: java.lang.ClassNotFoundException: net.sourceforge.cobertura.coveragedata.TouchCollector
  at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
  at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
  at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
  ... 51 more
```
3. Do it again in the same session:
```
scala> :paste
// Entering paste mode (ctrl-D to finish)


import io.archivesunleashed._
import io.archivesunleashed.matchbox._
val r = RecordLoader.loadArchives("/home/nruest/tmp/auk/75/3552/warcs/ARCHIVEIT-3552-DAILY-IEDLRH-20130212071008-00001-wbgrp-crawl052.us.archive.org-6680.warc.gz", sc)


// Exiting paste mode, now interpreting.

java.lang.NoClassDefFoundError: Could not initialize class io.archivesunleashed.package$RecordLoader$
  ... 57 elided
```

4. @edsu [example](https://gist.github.com/edsu/475ab8e2f3307a5854faaf7fd06a1c02)

**Expected behavior**

Well, this shouldn't happen. It should work.

**Environment information**
 - AUT version: 0.17.0
 - OS: Ubuntu 18.04
 - Java version: Java 8
 - Apache Spark version: Tested on 2.3.0, 2.3.2, 2.4.0
 - Apache Spark w/aut: `--jars`
 - Apache Spark command used to run AUT: `spark-shell --jars /home/nruest/Downloads/aut-0.17.0-fatjar.jar`

**Additional context**

@edsu notes in Slack that the `0.16.0` fatjar works.

--- a/pom.xml
+++ b/pom.xml
@@ -201,19 +201,6 @@
           </execution>
         </executions>
       </plugin>
-      <!-- for codecov.io -->
-      <plugin>
-        <groupId>org.codehaus.mojo</groupId>
-        <artifactId>cobertura-maven-plugin</artifactId>
-        <version>2.7</version>
-        <configuration>
-          <formats>
-            <format>html</format>
-            <format>xml</format>
-          </formats>
-          <check />
-        </configuration>
-      </plugin>
       <!-- for mycila -->
       <plugin>
         <groupId>com.mycila</groupId>
@@ -687,6 +674,19 @@
       <version>3.2</version>
     </dependency>
     <!--END issue #113-->
+    <!-- for codecov.io -->
+    <dependency>
+      <groupId>org.codehaus.mojo</groupId>
+      <artifactId>cobertura-maven-plugin</artifactId>
+      <version>2.7</version>
+      <scope>test</scope>
+      <exclusions>
+        <exclusion>
+          <groupId>ch.qos.logback</groupId>
+          <artifactId>logback-classic</artifactId>
+        </exclusion>
+      </exclusions>
+    </dependency>
   </dependencies>
 
   <developers>

